src/hypermedia-constants.js

```diff
@@ -582,12 +590,7 @@ export const Actions = {
 		updateFilesSubmissionLimit: 'update-files-submission-limit',
 		updateDefaultScoringRubric: 'update-default-scoring-rubric',
 		updateNotificationEmail: 'update-notification-email',
-		add: 'add',
-		categories: {
-			select: 'select',
-			deselect: 'deselect',
-			add: 'add'
-		}
+		add: 'add'
```

Comment:
duplicate of: https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/432/files#diff-b5ac48af05ca446b8b131d08b7a37fece756d6656a71b1d10d3d22920cd0bfa9R546 ?

Reply:
Good catch. Fixed!